cardano-node/src/Cardano/CLI/Benchmarking/Tx/NodeToNode.hs

```diff
@@ -177,11 +177,8 @@ instance Transformable Text IO (SendRecvTxSubmission ByronBlock) where
             -- Add a timestamp in 'ToObject'-representation.
             HM.insert "time" (String (T.pack . show $ currentTime)) obj
       tracer = if obj == emptyObject then nullTracer else tr
-    traceWith tracer =<<
-      LogObject
-        <$> pure mempty
-        <*> mkLOMeta (defineSeverity arg) (definePrivacyAnnotation arg)
-        <*> pure (LogStructured updatedObj)
+    meta <- mkLOMeta (defineSeverity arg) (definePrivacyAnnotation arg)
+    traceWith tracer (mempty, LogObject mempty meta (LogStructured updatedObj))
```

Comment:
This pattern seems to be pretty common. As I mentioned, it seems to be missing the newtype.
The second thing it might be missing is some constructor function that provides this "out-of-the-box".

Like `createNoContextLogObject :: MetaInfo -> LogStructured -> LogObject`.

Reply:
good point. yes, this can be further simplified as @deepfire already showed here.